### PR TITLE
Add cxxopts

### PR DESCRIPTION
**Pull request by SylvainCorlay**

### CODE REVIEW

**Comment by SylvainCorlay** [comment]
@gouarin as discussed. However 1.5 has not been tagged. I will update the repository when it is released.

**Comment by SylvainCorlay** [comment]
This is ready to go!

File: recipes/cxxopts/bld.bat [hunk 1]
```diff
@@ -0,0 +1,8 @@
+cmake -G "NMake Makefiles" -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% -D CXXOPTS_BUILD_EXAMPLES=OFF %SRC_DIR%
+if errorlevel 1 exit 1
+
+nmake
+if errorlevel 1 exit 1
+
```
Comment (ocefpaf) [inline recipes/cxxopts/bld.bat]: Is there a `ctest`/`make check` we can run?

File: recipes/cxxopts/build.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+cmake -DCMAKE_INSTALL_PREFIX=$PREFIX $SRC_DIR
+make install
```
Comment (ocefpaf) [inline recipes/cxxopts/build.sh]: `make check`?

**Pull request merged**